anvil/src/config.rs

```diff
@@ -69,7 +69,7 @@ const BANNER: &str = r#"
 #[derive(Debug, Clone)]
 pub struct NodeConfig {
     /// Chain ID of the EVM chain
-    pub chain_id: u64,
+    pub chain_id: Option<u64>,
```

Comment:
this must be an option because we need to distinguish if the chain_id was specified

Reply:
Yes correct